### PR TITLE
Update kitchen config

### PR DESCRIPTION
**Pull request by armiller**

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -14,38 +10,27 @@ provisioner:
...
+      packages:
+        - python27
     omnibus_updater:
       disabled: true
 
-platforms:
-  - name: centos-6.6
-    attributes:
-      mysql:
-        version: 5.1
```
Comment (rettigs) [inline .kitchen.yml]: Not sure what to do with this.

Comment (ramereth) [inline .kitchen.yml]: I tested this and its not needed anymore

**Comment by ramereth** [comment]
@osuosl-cookbooks/chefs ready for review

**Comment by armiller** [comment]
:+1:

**Pull request merged**